### PR TITLE
Update Files.it-IT.xlf

### PR DESCRIPTION
**Pull request by SimoneRaspagni**
Update string translation

### CODE REVIEW

**Comment by xPoppyx** [comment]
all states must be in: state = "translated"

File: Files/MultilingualResources/Files.it-IT.xlf [hunk 1]
```diff
@@ -144,7 +144,7 @@
         </trans-unit>
         <trans-unit id="StatusBarControlInvertSelection.Text" translate="yes" xml:space="preserve">
           <source>Invert Selection</source>
-          <target state="new">Invert Selection</target>
+          <target state="new">Inverti selezione</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.it-IT.xlf:147]: Please change the state from `new` to `translated`.

Comment (SimoneRaspagni) [reply Files/MultilingualResources/Files.it-IT.xlf:147]: Got it

Comment (yaichenbaum) [reply Files/MultilingualResources/Files.it-IT.xlf:147]: It looks like the state for "Invert Selection" still needs to be updated.

**Pull request merged**